### PR TITLE
Fix typo in the Introduction of the configuration page

### PR DESCRIPTION
**Pull request by Caetano-Lopes**
** Adds to #9 **

### CODE REVIEW

File: docs/_docs/configuration/configuration.md [hunk 1]
```diff
@@ -22,8 +22,8 @@ has_children: true
 
 ## Introduction
 
-Laravel Translate strives to be as customisable as possible, so it can be used in any situation. The two main ways to customise the package are through the service provider and the configuration file.
+Laravel Translate strives to be as customizable as possible, so it can be used in any situation. The two main ways to customize the package are through the service provider and the configuration file.
```
Comment (tobytwigger) [inline docs/_docs/configuration/configuration.md:25]: These were spelt correctly (or at least the English version). I think I'm happy to keep it as an s not a z!

**Comment by tobytwigger** [comment]
@Caetano-Lopes Thanks for this! I've left you a comment.

Have you reviewed all the documentation? If not, would you be able to change your pull request description to say ** Adds to #9 ** so we still review the other pages.

**Comment by Caetano-Lopes** [comment]
> @Caetano-Lopes Thanks for this! I've left you a comment.
> 
> Have you reviewed all the documentation? If not, would you be able to change your pull request description to say ** Adds to #9 ** so we still review the other pages.
> 
> Could you also change the pull request to be against the dev branch, as we don't usually accept changes to master.

I reverted what you requested and reviewed all the documents. Check if it helps! Cheers :D

**Comment by tobytwigger** [comment]
Thanks a lot for your help, hope you get some use out of this package!

**Pull request merged**